internal/daemon/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -862,6 +862,7 @@ func toAuthMethodProto(ctx context.Context, in auth.AuthMethod, opt ...handlers.
 			AllowedAudiences:  i.GetAudClaims(),
 			ClaimsScopes:      i.GetClaimsScopes(),
 			AccountClaimMaps:  i.GetAccountClaimMaps(),
+			Prompts:           i.GetPrompts(),
```

Comment:
sort of a general question: should we add
prompts to the appropriate test in authmethod_service_test.go?